frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelinesGlobal.cy.ts

```diff
@@ -116,9 +109,104 @@ describe('Pipelines Global', () => {
     cy.wait('@refreshVersions');
 
     // Verify the uploaded pipeline version is in the table
-    pipelinesTable.toggleExpandRowByIndex(0);
+    pipelinesTable.toggleExpandRowByName(initialMockPipeline.name);
     pipelinesTable.findRowByName('New pipeline version');
   });
+
+  it.only('delete pipeline and versions', () => {
```

Comment:
Since there are different modals shown for single and bulk deletions for pipelines and versions, I think it would be best to have separate tests for these operations.